org.gnome.Photos.json

```diff
@@ -103,22 +104,11 @@
                 }
             ]
         },
-        {
```

Comment:
Why this change? especially that you re-add it later.

Reply:
Based on https://gitlab.gnome.org/danigm/gnome-photos/-/commit/bfbf250252bb93db02a9866f3c1468e9b2386a11. I don't think it changes much, but why risk it.